docs/maintain/maintain-sync.md

```diff
@@ -101,10 +101,14 @@ example:
 
 :::
 
-<Tabs groupId="operating-systems" values={[ {label: 'macOS', value: 'mac'}, {label: 'Windows',
-value: 'win'}, {label: 'Linux (standalone)', value: 'linux-standalone'}, {label: 'Linux (package)',
-value: 'linux-package'}, ]}
-
+<Tabs
+  groupId="operating-systems"
+  values={[
+    {label: 'macOS', value: 'mac'},
+    {label: 'Windows', value: 'win'},
+    {label: 'Linux (standalone)', value: 'linux-standalone'},
+    {label: 'Linux (package)', value: 'linux-package'},
+  ]}
 >
```

Comment:
@DrW3RK This seems to resolve it, if the closing `>` on `Tabs` isn't placed on a newline.  Otherwise, it reverts it the incorrect formatting.

```suggestion
<Tabs groupId="operating-systems" values={[ {label: 'macOS', value: 'mac'}, {label: 'Windows',
value: 'win'}, {label: 'Linux (standalone)', value: 'linux-standalone'}, {label: 'Linux (package)',
value: 'linux-package'} ]}>

<TabItem value="mac">
```